cli/internal/providers/datacatalog/localcatalog/catalog.go

```diff
@@ -21,6 +21,8 @@ const (
 	KindCategories    = "categories"
 	KindTrackingPlans = "tp"
 	KindCustomTypes   = "custom-types"
+
+	KindTrackingPlanForReference = "tracking-plans"
```

Comment:
The reason this change is to facilitate the reference generation of the trackingplans imported correctly.

It was getting generated as `#/tp/<metadata-name>/:tpid` and now it will be `#/tracking-plans/<metadata-name>/:tpid`